Modules/ITS/include/ITS/ITSFhrTask.h

```diff
@@ -172,12 +172,8 @@ class ITSFhrTask final : public TaskInterface
   TH2I* mChipStaveEventHitCheck[7];
   TH1D* mOccupancyPlot[7];
 
-  std::string mRunNumberPath;
-  std::string mRunNumber = "000000";
-
   // Geometry decoder
   o2::its::GeometryTGeo* mGeom;
-  std::string mGeomPath;
```

Comment:
@zz951 can you please re-introduce the geomPath both here and in the json? Please take a look at what is in the QC master:   https://github.com/AliceO2Group/QualityControl/blob/1a77eacf17f13d1e403391b9f3a4a23282c0c192/Modules/ITS/src/ITSFhrTask.cxx#L96